keps/sig-instrumentation/2305-metrics-cardinality-enforcement/README.md

```diff
@@ -236,19 +264,145 @@ This would then be interpreted by our machinery as this:
 
 ```
 
-## Open-Question
-
+## Design Details
+### Test Plan
+Unit test to verify that the metric label will be set to "unexpected" if the metric encounters label values outside our explicit allowlist of values.
```

Comment:
What would we set the metric value to in this case? Should it be emitted at all?

Reply:
I don't know if I get your question. The value will be set to "unexpected" if it is out of allow-list.

Reply:
We need to emit something otherwise our aggregation counts will be off.

Reply:
Makes sense @logicalhan - we'll just aggregate stats into that last bucket? Totals and durations values etc. will be unaffected?

Reply:
Yes, that's how it works in the manual cases in the k8s codebase where we do this today.